pkg/controllers/management/oidcprovider/controller.go

```diff
@@ -16,7 +17,7 @@ import (
 	"github.com/rancher/rancher/pkg/wrangler"
 	corev1 "github.com/rancher/wrangler/v3/pkg/generated/controllers/core/v1"
 	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
This is usually imported as `apierrors` might be useful to retain the naming for readers?

* https://github.com/rancher/rancher/blob/main/pkg/controllers/managementagent/dnsrecord/dnsrecord.go#L20
* https://github.com/rancher/rancher/blob/main/pkg/controllers/management/auth/globalroles/globalrole_handler.go#L18